Enhancement/issue62 Support for importing a subpath of a remote repo in create repo script (#75)
devonfw/hangar: documentation/azure-devops/setup-repository-script.asciidoc

> -r, --remove-other-branches            When combined with -b (and possibly -s), removes any other remaining branch.

Reviewer: ```suggestion
-r, --remove-other-branches            Removes branches other than the one specified in -b (or default one).

```

-s, --setup-branch-strategy            Creates branches and policies required for the desired workflow. Requires -b on import. Accepted values: gitflow.
-f, --force                            Skips any user confirmation.

>     --subpath                          When importing from an URL, will initialize the repo with the subpath set. (ignored if -g, -b or -r not set)

Reviewer: ```suggestion
    --subpath                          When combined with -g and -r, imports only the specified subpath of the source Git repository.

```
